dts/bindings/qspi/st,stm32-qspi.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020, Linaro limited
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    STM32 QSPI device representation. A stm32 quadspi node would typically
+    looks to this:
+
+        &quadspi {
+            pinctrl-0 = <&quadspi_clk_pe10 &quadspi_ncs_pe11
+                         &quadspi_bk1_io0_pe12 &quadspi_bk1_io1_pe13
+                         &quadspi_bk1_io2_pe14 &quadspi_bk1_io3_pe15>;
+
+            dmas = <&dma1 5 5 0x0000 0x03>;
+            dma-names = "tx_rx";
+
+            status = "okay";
+        };
+
+compatible: "st,stm32-qspi"
+
+include: base.yaml
+
+bus: qspi
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    dmas:
+      required: true
+      description: |
+        DMA channel specifier. Specifier will have a phandle
+        reference to the dma controller, the channel number, the slot number,
+        channel configuration and finally features.
+
+        For example dmas for TX/RX on QSPI
+           dmas = <&dma1 5 5 0x0000 0x03>;
+
+    dma-names:
+      required: true
+      description: |
+        DMA channel name. Expected value "tx_rx".
```

Comment:
Can this be anything else?  When?  If it can't, maybe it should be `const` and not `required`.

Not blocking.

Reply:
I'm not following. Is the comment still valid in the light of the last commits?

Reply:
Eh, well, the comment is here because I didn't *see* the last commits that changed it: I like to review one commit at a time.

I'm not requiring any changes, just pointing out something that confused me.  I'll take this as "too much hassle".

Reply:
> I like to review one commit at a time.

You're fully right. I usually don't like this kind of split in reviews neither.